nvflare/private/fed/server/cmd_utils.py

```diff
@@ -91,7 +92,17 @@ def validate_command_targets(self, conn: Connection, args: List[str]) -> str:
             # get all clients
             clients = engine.get_clients()
         else:
+            # check whether client names contain admin clients.
+            admin_clients = []
+            for c in client_names:
+                if is_valid_admin_client_name(c):
+                    admin_clients.append(c)
+
             clients, invalid_inputs = engine.validate_targets(client_names)
+            if admin_clients:
+                # admin clients are considered invalid since admin commands do not go to admin clients!
+                invalid_inputs.extend(admin_clients)
```

Comment:
should we instead modify the "validate_targets" of "engine" to return admin_clients as "invalid_inputs"?